### PR TITLE
Add board:  Nebula S3

### PR DESCRIPTION
**Pull request by DocMonster7**
## Description 
A variant of ESP32-S3 development board created with sole focus on hobbyist with on board IMU which is MPU-6050 and 
WS2812B LED with Battery connector. 

## Schematic 

Schematic_Nebula V1_2023-05-26

## Boards

Nebula s3

### CODE REVIEW

File: boards.txt [hunk 1]
```diff
@@ -23243,3 +23243,225 @@ crabik_slot_esp32_s3.menu.EraseFlash.all=Enabled
...
+nebulas3.serial.disableDTR=false
+nebulas3.serial.disableRTS=false
+
+nebulas3.build.tarch=xtensa
+nebulas3.build.bootloader_addr=0x0
+nebulas3.build.target=esp32s3
+nebulas3.build.mcu=esp32s3
+nebulas3.build.core=esp32
+nebulas3.build.variant=Nebula_S3
+nebulas3.build.board=ESP32S3_DEV
```
Comment (P-R-O-C-H-Y) [inline boards.txt:23275]: ESP32S3_DEV should be changed to some different name like NEBULAS3.

File: boards.txt [hunk 2]
```diff
@@ -23243,3 +23243,225 @@ crabik_slot_esp32_s3.menu.EraseFlash.all=Enabled
...
+nebulas3.menu.FlashMode.opi.build.flash_freq=80m
+
+nebulas3.menu.FlashSize.4M=4MB (32Mb)
+nebulas3.menu.FlashSize.4M.build.flash_size=4MB
+nebulas3.menu.FlashSize.8M=8MB (64Mb)
+nebulas3.menu.FlashSize.8M.build.flash_size=8MB
+nebulas3.menu.FlashSize.8M.build.partitions=default_8MB
+nebulas3.menu.FlashSize.16M=16MB (128Mb)
+nebulas3.menu.FlashSize.16M.build.flash_size=16MB
+
```
Comment (P-R-O-C-H-Y) [inline boards.txt:23345]: please just remove this empty line, which is doubled.

**Review by P-R-O-C-H-Y — CHANGES_REQUESTED** [review]
@DocMonster7 PTAL on my comment.

**Comment by DocMonster7** [comment]
@P-R-O-C-H-Y Please review changes made.

**Comment by P-R-O-C-H-Y** [comment]
@DocMonster7 It seems you kind of changed whole boards.txt file. Can't load the diffs.
_23,467 additions, 23,467 deletions not shown because the diff is too large. Please use a local Git client to view these changes._

**Comment by DocMonster7** [comment]
Sorry about that, I will try to resolve it using my local client and then push the changes instead of the web editor

**Comment by P-R-O-C-H-Y** [comment]
Looks good now, but please change the board to UPPERCASE -> NebulaS3 to NEBULAS3. Thanks

**Comment by DocMonster7** [comment]
I have made the requested changes. Have a look.

**Review by P-R-O-C-H-Y — APPROVED** [review]
All good now :)

**Pull request merged**